editoast/src/core/mq_client.rs

```diff
@@ -51,6 +49,12 @@ pub enum Error {
     #[error("Connection does not exist")]
     #[editoast_error(status = "500")]
     ConnectionDoesNotExist,
```

Comment:
nit: "Connection" is not so expressive here (maybe other errors are candidate to this).
How about having the same suffix? (nit: Not sure `Lapin` is the most expressive)
```suggestion
    ConnectionDoesNotExistLapin,
```

Reply:
Will do in another PR, didn't change the name of this one while working on this PR